lib/db/DB.ts

```diff
@@ -112,8 +119,13 @@ class DB {
     try {
       this.sequelize = new Sequelize('', this.sequelize.options.username, this.sequelize.options.password, this.sequelize.options);
       await this.sequelize.authenticate();
-      await this.sequelize.query(`CREATE DATABASE ${this.sequelize.options.database};`);
+      await this.sequelize.query(`CREATE DATABASE ${this.sequelize.options.database} CHARACTER SET utf8 COLLATE utf8_general_ci;`);
       await this.sequelize.query(`USE ${this.sequelize.options.database}`);
+      this.close();
+
+      // If the models aren't reinitialized the integration test fails
+      this.initSequelize();
```

Comment:
this doesn't seems to solve the problem when I test it on my env.
are you sure it makes your `test:int` pass on the first run, and not the second one?

Reply:
Did you run `npm run compile` after checking out my branch before you ran `npm run test:int`?

Because it works just fine here :thinking:

Reply:
you're right ;)
I didn't ran your branch properly. Now when I did, it works.

Thanks for finding the problem & the solution. We're gonna refactor this whole area (DB, tests, gulp), but it's good to have something which is already working.